css-positioning/fixed.css

```diff
@@ -0,0 +1,10 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  left: 500px;
```

Comment:
Works in this case, but it wouldn't be dynamic for all screen sizes. 500px on my screen is different than yours. You can utilize both `left` & `right` to fix that

Reply:
And would you use percentages with that too?